src/components/BubbleSort.js

```diff
@@ -0,0 +1,79 @@
+export default class BubbleSort {
+  constructor(elements) {
+    this.numberElements = elements;
+  }
+
+  sortingElements() {
+    const elements = this.numberElements;
+    const l = elements.length;
```

Comment:
1. powinno się unikać nazw zmiennych jak "l". const length będzie ok. 
2. Skąd masz pewność że elements to jest tablica? Na początku funkcji trzeba sprawdzić czy przekazany typ danych jest poprawny. JS to język dynamicznie typowany więc trzeba samemu zadbać o to żeby się wszystko zgadzało.